lineage/figures/figure5.py

```diff
@@ -16,6 +16,11 @@
     commonAnalyze
 )
 from ..LineageTree import LineageTree
+from ..states.StateDistributionGamma import StateDistribution
+
+pi2 = [1]
+T2 = np.array([1])
+E3 = [StateDistribution(0.99, 8, 6)]
 
```

Comment:
replace them with:
`pi = np.ones(len(E3)) / len(E3)
T = np.eye(len(E3)`
and define E above them.

Reply:
make sure to `git pull`